### PR TITLE
Add payload formatters doc

### LINKED ISSUE #2713: Document payload formatters
#### Summary

Document payload formatters.

#### Why do we need this ?

We provide several options for the users on how device payload are decoded/encoded. 

#### What is already there? What do you see now?

Functionality to set formatters in the Console

Screenshot 2020-06-11 at 17 24 55

Screenshot 2020-06-11 at 17 23 41

It is also possible to set payload formatters via the CLI

#### What is missing? What do you want to see?

1. Describe each formatter (javascript, grpc, cayennelpp and repository).
	* Having some examples would be nice.
	* for `javascript` it would be great to mention what js runtime we use in the AS, so that the users dont try using unavailable js API, cc @johanstokking 
	* for `repository` mention that it is up to device manufacturers and it might be available.
2. Describe differences between device level and application level formatters. 
	* Why would one use application level formatters?
	* How to reset device level formatters and fallback to formatters in the application link?
3. Describe how to set formatters via the Console and CLI
	

We might want to wait until https://github.com/TheThingsNetwork/lorawan-stack/issues/2234 is implemented to describe how one can quickly test their formatters.
#### Can you do this yourself and submit a Pull Request?

cc @benolayinka @nejraselimovic

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Closes #2713 

#### Changes

- Add payload formatter docs for Javascript, CayenneLPP, and Device Repo

#### Testing

Run locally

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by johanstokking** [comment]
Don't know what happened here but I saw a merge conflict in the diff. Rebased and pushed.

**Comment by benolayinka** [comment]
@bafonins can you take a look before I merge

File: doc/content/integrations/payload-formatters/grpc.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "GRPC"
+description: ""
+draft: true
+---
+
+GRPC Payload Formatters are not yet implemented.
```
Comment (bafonins) [inline doc/content/integrations/payload-formatters/grpc.md:7]: I guess we should display a notification in the console to warn the users that this option is not currently supported, or to hide this option.

@johanstokking ?

Comment (johanstokking) [reply doc/content/integrations/payload-formatters/grpc.md]: I wouldn't even mention this in the docs

**Pull request merged**